storage-transfer/src/main/java/com/google/cloud/storage/storagetransfer/samples/TransferFromAzure.java

```diff
@@ -0,0 +1,107 @@
+import com.google.storagetransfer.v1.proto.TransferTypes.AzureCredentials;
+import com.google.storagetransfer.v1.proto.TransferTypes.GcsData;
+import com.google.storagetransfer.v1.proto.TransferTypes.TransferJob;
+import com.google.storagetransfer.v1.proto.TransferTypes.TransferJob.Status;
+import com.google.storagetransfer.v1.proto.TransferTypes.TransferSpec;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class TransferFromAzure {
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    // Your Google Cloud Project ID
+    String projectId = "my-project-id";
+
+    // Your Azure Storage Account name
+    String azureStorageAccountName = "myAzureAccountName";
+
+    // The Azure source container to transfer data from
+    String azureSourceContainer = "my-source-container";
+
+    // The GCS bucket to transfer data to
+    String gcsSinkBukcet = "my-sink-bucket";
+
+    transferFromAzureBlobStorage(
+        projectId, azureStorageAccountName, azureSourceContainer, gcsSinkBukcet);
+  }
+
+  public static void transferFromAzureBlobStorage(String projectId, String azureStorageAccountName,
+      String azureSourceContainer, String gcsSinkBucket)
+      throws IOException, ExecutionException, InterruptedException {
+
+    // Your Azure SAS token, should be accessed via environment variable
+    String azureSasToken = System.getenv("AZURE_SAS_TOKEN");
+
+    TransferJob transferJob =
+        TransferJob.newBuilder()
+            .setProjectId(projectId)
+            .setStatus(Status.ENABLED)
+            .setTransferSpec(
+                TransferSpec.newBuilder()
+                    .setAzureBlobStorageDataSource(
+                        AzureBlobStorageData.newBuilder()
+                            .setAzureCredentials(AzureCredentials.newBuilder()
+                            .setSasToken(azureSasToken).build())
+                            .setContainer(azureSourceContainer)
+                            .setStorageAccount(azureStorageAccountName))
+                    .setGcsDataSink(GcsData.newBuilder().setBucketName(gcsSinkBucket).build())
+                    .build())
+            .build();
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources,
+    // or use "try-with-close" statement to do this automatically.
+    try (StorageTransferServiceClient storageTransfer = StorageTransferServiceClient.create()) {
+      // Create the transfer job
+      TransferJob response =
+          storageTransfer.createTransferJob(
+              TransferProto.CreateTransferJobRequest.newBuilder()
```

Comment:
Suggesting to inline the "TransferProto" to make it more readable.

Reply:
Apologies if I wasn't clear enough. Could you include ```TransferProto.CreateTransferJobRequest``` in the import statement so that you could directly access ```CreateTransferJobRequest.newBuilder()``` in this line?